src/block/precomputed.rs

```diff
@@ -15,12 +17,14 @@ pub struct BlockLogContents {
     pub(crate) state_hash: String,
     pub(crate) blockchain_length: Option<u32>,
     pub(crate) contents: Vec<u8>,
+    pub(crate) global_slot_since_genesis: GlobalSlotNumberV1,
```

Comment:
We don't need to add another field to `BlockLogContents`. The `contents` field holds all of the bytes in the file.